rollup.config.js

```diff
@@ -1,40 +1,39 @@
-import babel from 'rollup-plugin-babel'
-import builtins from 'rollup-plugin-node-builtins'
-import resolve from 'rollup-plugin-node-resolve'
-import commonjs from '@rollup/plugin-commonjs'
-import json from '@rollup/plugin-json'
-import html from '@rollup/plugin-html'
-import cleaner from 'rollup-plugin-cleaner'
-import { template } from './example/src/template.js'
-import typescript from '@rollup/plugin-typescript'
+import babel from 'rollup-plugin-babel';
+import builtins from 'rollup-plugin-node-builtins';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import html from '@rollup/plugin-html';
+import cleaner from 'rollup-plugin-cleaner';
+import { template } from './example/src/template.js';
+import typescript from '@rollup/plugin-typescript';
+
```

Comment:
this whitespace seems superfluous

Reply:
agreed - sorry + fixed

Reply:
This is just ";" missing from end of lines.

![image](https://user-images.githubusercontent.com/1444266/208416581-4adbd3b5-7e30-48f1-9954-d92979d92dd4.png)

Reply:
removed from PR